src/app/(app)/search-result/[id]/page.tsx

```diff
@@ -0,0 +1,11 @@
+import SearchResultDetail from '@/components/SearchResultDetail/SearchResultDetail';
+
+export default function SearchResultDetailPage({ params }: { params: { id: string } }) {
```

Comment:
The suffix `Page` is inconsistent with the other page so could be omitted:

```suggestion
export default function SearchResultDetail({ params }: { params: { id: string } }) {
```